serverless.yml

```diff
@@ -71,6 +75,7 @@ custom:
   s3:
     host: localhost
     directory: /tmp
+    buckets: hathor-explorer-metadata-dev
```

Comment:
dev? Why not `hathor-explorer-metadata`?

Reply:
This was changed on [e5ba6ac](https://github.com/HathorNetwork/hathor-explorer-service/pull/174/commits/e5ba6ac75a5a4fb833c567cc07543627583b3439) to use the `METADATA_BUCKET` environment variable, changing with the run context.